barretenberg/cpp/src/barretenberg/benchmark/goblin_bench/eccvm.bench.cpp

```diff
@@ -65,8 +65,8 @@ void eccvm_prove(State& state) noexcept
     };
 }
 
-BENCHMARK(eccvm_generate_prover)->Unit(kMillisecond)->DenseRange(10, 20);
-BENCHMARK(eccvm_prove)->Unit(kMillisecond)->DenseRange(10, 20);
+BENCHMARK(eccvm_generate_prover)->Unit(kMillisecond)->DenseRange(12, 18);
```

Comment:
The proving is quite slow and the time grows somewhat predictably.

Reply:
sounds good. 2^20 is a lot of eccvm operations, far more than we expect on the client, so reducing the range is fine